orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -135,3 +160,30 @@ def _replicate_for_data_parallel(self):
 
         raise NotImplementedError("ORTModule is not compatible with torch.nn.DataParallel. "
                                   "Please use torch.nn.parallel.DistributedDataParallel instead.")
+
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
+                                missing_keys, unexpected_keys, error_msgs):
+        """Override original method to delegate execution to the original PyTorch user module"""
+
+        self._module.original_module()._load_from_state_dict(state_dict, prefix, local_metadata, strict,
+                                missing_keys, unexpected_keys, error_msgs)
+
+    def named_children(self) -> Iterator[Tuple[str, 'Module']]:
+        """Override original method to delegate execution to the original PyTorch user module"""
+
+        yield from self._module.original_module().named_children()
+
+    def modules(self) -> Iterator['Module']:
+        """Override original method to delegate execution to the original PyTorch user module"""
+
+        yield from self._module.original_module().modules()
+
+    def named_modules(self, memo: Optional[Set['Module']] = None, prefix: str = ''):
+        """Override original method to delegate execution to the original PyTorch user module"""
+
+        yield from self._module.original_module().named_modules(memo, prefix)
+
+    def add_module(self, name: str, module: Optional['Module']) -> None:
+        """Override original method to delegate execution to the original PyTorch user module"""
+
+        self._module.original_module().add_module(name, module)
```

Comment:
Do we actually support `add_module()`? I assume we don't support this after `__init__()`, and it seems like something that should raise `NotImplementedError`.